learning_journal/templates/detail.jinja2

```diff
@@ -3,11 +3,10 @@
         <main>
             <article class="post">
                 <h3>{{ entry.title }}</h3>
-                    <h4>{{ entry.creation_date }}</h4>
                     <p>{{ entry.body }}</p>
```

Comment:
Change the indentation this \<p> block one step back (at the same indent level as \<h3>